lms/djangoapps/teams/views.py

```diff
@@ -313,8 +313,14 @@ def get(self, request):
             )
         else:
             return Response({
-                'developer_message': "unsupported order_by value {}".format(order_by_input),
-                'user_message': _(u"The ordering {} is not supported").format(order_by_input),
+                'developer_message': "unsupported order_by value {ordering}".format(ordering=order_by_input),
+                # Translators: 'ordering' describes how to order the
+                # list of teams returned from this view. For example,
+                # 'lower_name' indicates that the list should be
+                # supported by the team name in lower case, and
+                # 'team_size' will sort by the number of members per
+                # team.
```

Comment:
OK now that I have the context, this may be too much detail. Translators are typically non-technical (at least ,non-programmers). A better way to word this would be:

Translators: "ordering" is a string describing a way of ordering a list. For example, `{ordering}` may be 'lower_case', indicating that the user had wanted to order the list by lower case name.

If there's a better more straightforward example than "lower_case", I'd use that (eg is "name" an option? order by name? or date, order by date? Name and date are intuitive and are very general ways of ordering across purposes, not necessarily related to the Teams feature). Having your translators context too specific to your use case will only confuse translators who are actually not familiar with the feature, and Teams is a great example of a feature that's very difficult for the majority of users to access.